### PR TITLE
data:B2prakash

### PR DESCRIPTION
**Pull request by B2prakash**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

you'll are doing a great job . i would like my linkfree profile to be updated 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5563"></a>

### CODE REVIEW

**Review by Krishnapro — APPROVED** [review]
Look good to merge

**Review by khushalarora — DISMISSED** [review]
Thank you for contributing  :tada:

File: data/B2prakash.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      "group": "Primary Socials",
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/VedPrak39212943",
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
+    },
+    {
+      "group": "Primary Socials",
+      "name": "LinkedIn: Connect with  me",
+      "url": "https://linkedin.com/in/ved prakash",
```
Comment (kumarsonsoff3) [inline data/B2prakash.json:29]: There is a broken URL, Please update it with your LinkedIn url.

File: data/B2prakash.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      "icon": "FaLinkedin",
+      "color": "#00ACEE"
+    },
+    {
+      "group": "Primary Socials",
+      "name": "YouTube: Learn more about Open Source",
+      "url": "https://youtube.com/@ved prakash",
```
Comment (kumarsonsoff3) [inline data/B2prakash.json:36]: Please fix this one too, as it takes me to `404` 🛠

File: data/B2prakash.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    {
+      "group": "Primary Socials",
+      "name": "GitHub: Collaborate on Open Source",
+      "url": "https://github.com/B2prakash",
+      "icon": "FaGithub"
+    },
+    {
+      "group":"Support",
+      "name": "Donate: GitHub Sponsors",
+      "url": "https://github.com/sponsors/eddiejaoude",
```
Comment (kumarsonsoff3) [inline data/B2prakash.json:49]: You might want to update it with your Sponsor URL. If you don't have a sponsor page. Then you can remove this section.

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/B2prakash.json (hunk 2 above)
Comment (B2prakash) [inline data/B2prakash.json:36]:
```suggestion
      "url": "https://youtube.com/@vedprakash2431",
```

File: data/B2prakash.json (hunk 1 above)
Comment (B2prakash) [inline data/B2prakash.json:29]:
```suggestion
      "url": "https://linkedin.com/in/vedprakash",
```

File: data/B2prakash.json [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+      "icon": "FaRegMoneyBillAlt"
+    }
```
Comment (B2prakash) [inline data/B2prakash.json:51]:
```suggestion
   
```

File: data/B2prakash.json [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+      "url": "https://youtube.com/@vedprakash2431",
+      "icon": "FaYoutube",
+      "color": "#FF0000"
+    },
+    {
+      "group": "Primary Socials",
+      "name": "GitHub: Collaborate on Open Source",
+      "url": "https://github.com/B2prakash",
+      "icon": "FaGithub"
+    },
```
Comment (B2prakash) [inline data/B2prakash.json:45]:
```suggestion
    }
```

**Comment by eddiejaoude** [comment]
Did you mean to close this @B2prakash ?

**Pull request closed without merging**